### DISCUSSION TITLE
Agents now have stopped working on some computers which are being added.

### DISCUSSION THREAD

**Discussion by rubybrars** [amidaware/tacticalrmm#1744] [General]
**Server Info (please complete the following information):**
 - OS: [e.g. Ubuntu 20.04, Debian 10]Debian 11
 - Browser: [e.g. chrome, safari]all
 - RMM Version (as shown in top left of web UI) :0.17.1

**Installation Method:**
  - [ ] Standard

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI):
- Agent OS: [e.g. Win 11]

**Describe the bug**
Some of the agents are not getting info after initial install right away and say last booted 54 years ago.
Most of the steps on these agents dont work
I have attached a picture as well.

**Screenshots**

**Additional context**
![image](https://github.com/amidaware/tacticalrmm/assets/140012835/ccdef6b7-0f07-4285-8234-715c9a279498)
![RmmError](https://github.com/amidaware/tacticalrmm/assets/140012835/3ba99c9f-ff4d-4000-8a8f-33871dc1b70f)

**Comment by dinger1986** [comment]
When did you first install? Are you using a proxy or any other custom settings?

**Reply by rubybrars** [reply]
We got about 260 agents, out of which 217 agents are windows desktops.
RMM was installed last year in June i believe on debian 11
Since then it was all working ok, up until few weeks ago.
Now some new agents when installed are showing last rebooted 54 yeas ago, and those particular agents cannot perform tasks.
However take control works no problem.

**Reply by dinger1986** [reply]
Have you renewed your certificate?

**Reply by wh1te909** [reply]
run `./update.sh --force` this should fix issues, then reinstall those agents showing 54 years.

**Reply by rubybrars** [reply]
I just updated the server to latest 17.3 and all agents lost connectivity

**Reply by rubybrars** [reply]
all agents show offline now. Got alert emails saying everything is down

**Reply by rubybrars** [reply]
The agent count on the web is showing 0 as well, so nothing

**Reply by rubybrars** [reply]
mesh agents are working and showing online and a count too, rmm web interface is showing agents but no live and saying count is 0

![image](https://github.com/amidaware/tacticalrmm/assets/140012835/4738a4ed-23e4-4d97-ad3a-afa2558bbc83)

**Reply by wh1te909** [reply]
what's the output of `sudo systemctl status --full --no-pager nats nats-api`

**Reply by rubybrars** [reply]
![image](https://github.com/amidaware/tacticalrmm/assets/140012835/ca6db447-b807-4de6-9dd4-77b1ae6d56cc)

**Reply by wh1te909** [reply]
it tells you the error there. if you're using custom certs you need to follow these instructions: https://docs.tacticalrmm.com/functions/settings_override/#using-your-own-wildcard-ssl-cert

**Reply by rubybrars** [reply]
thx
